include/seqan3/alignment/matrix/detail/alignment_score_matrix_one_column_base.hpp

```diff
@@ -39,8 +39,10 @@ struct alignment_score_matrix_one_column_base
     static_assert(arithmetic<score_t> || simd_concept<score_t>,
                   "Score type must either be either an arithmetic type or a simd vector type.");
 
+    //!\brief The underlying type of the scores.
+    using underlying_type = score_t;
```

Comment:
But it is used for other matrices whose type is not a score type as well

Reply:
If these other matrices are not score matrices they shouldn't inherit from this class...
Or do you mean that `underlying_type` is a concept requirement for any matrix type?

Reply:
It is not about inheritance. It is about a conceptual design. Note, that there is no concept for this yet, since the types are solely used inside of the alignment but it is a generic name that is alike for all these martices. So if you will, yes they fulfill kind of a concept.

Reply:
Thanks, that's fine then!